Modules/repetition.js

```diff
@@ -0,0 +1,28 @@
+// eslint-disable-next-line import/no-cycle
+import { Book } from './book.js';
+
+const msg = document.querySelector('#errorMsg');
+const deleteBtn = document.createElement('button');
+
+// eslint-disable-next-line import/prefer-default-export
```

Comment:
[OPTIONAL]
- I noticed that you're exporting just one function from this module. What do you think about using the `default` export instead of the named export? To do this, kindly remove the `eslint-disable` statement and add `default` to the export statement on line 8. You can get more information from this [article](https://plmlab.math.cnrs.fr/tbrousso/imt_maquette_reunionv4/-/blob/master/node_modules/eslint-plugin-import/docs/rules/prefer-default-export.md). I hope it helps 😀